bedrock/firefox/templates/firefox/dev-whatsnew.html

```diff
@@ -68,49 +81,18 @@
           <a href="https://hacks.mozilla.org/?p=28808" rel="external" class="more">{{_('See how you helped')}}</a>
```

Comment:
I checked the link and this doesn't seem right (it's about fx40).

@ckprice Are we exposing the whatsnew page at all to localizations? How can I know? I was preparing this page and realized we never translated this blurb (added in [this bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1136912)).